### DISCUSSION TITLE
How to fix this type reference?

### DISCUSSION THREAD

**Discussion by pedroschmid** [WhiskeySockets/Baileys#151] [General]
src/wa.ts:154:41 - error TS2345: Argument of type '{ get: (type: keyof SignalDataTypeMap, ids: string[]) => Promise<{ [key: string]: Uint8Array | KeyPair | { [jid: string]: boolean; } | IAppStateSyncKeyData | LTHashState; }>; set: (data: any) => Promise<...>; }' is not assignable to parameter of type 'SignalKeyStore'.

**Comment by Marcoant007** [comment]
You have to update baileys store, use this lib that will work just fine
https://www.npmjs.com/package/mrx-baileys-store?activeTab=readme
